src/App.tsx

```diff
@@ -1,24 +1,195 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { Notification } from './components/Notification/Notification';
+
+import { Todo } from './types/Todo';
+
+import {
+  getTodos, addTodo, onDelete, onUpdate,
+} from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 6476;
+
+export enum SortType {
+  ALL = 'All',
+  ACTIVE = 'Active',
+  COMPLETED = 'Completed',
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState('All');
```

Comment:
```suggestion
  const [filterBy, setFilterBy] = useState(SortType.ALL);
```

Reply:
I used `string` cause am creating nav dynamically

Reply:
`SortType.ALL` is the same `All` (i.e value from `enum` object) string, but you should use it instead of the common `All` string